project-management-infrastructure/src/main/java/life/qbic/controlling/infrastructure/sample/SamplePreviewJpaRepository.java

```diff
@@ -98,7 +97,7 @@ public static Specification<SamplePreview> isDistinct() {
     //If no filter was provided return all SamplePreviews
     public static Specification<SamplePreview> isBlank(String filter) {
       return (root, query, builder) -> {
-        if (StringUtil.isBlank(filter)) {
+        if (filter.isBlank()) {
```

Comment:
Good spot! However, StringUtil::isBlank checks for null as well according to the JavaDoc.
If the filter is null you would produce a NullPointer with String::isBlank